src/utils/ocean.ts

```diff
@@ -33,15 +20,14 @@ export function getOceanConfig(network: string | number): ConfigHelperConfig {
       : process.env.GATSBY_INFURA_PROJECT_ID
   ) as ConfigHelperConfig
 
-  return network === 'gaiaxtestnet' || network === 2021000
-    ? {
-        ...config,
-        ...gaiaxtestnetConfig
-      }
-    : network === 'catenaxtestnet' || network === 2021001
+  const configOverwrite = Object.entries(
+    chains as { [name: string]: ConfigHelperConfigOverwrite }
+  ).find(([alias, chainConfig]) => chainConfig.networkId === config.networkId)
```

Comment:
We need to adjust this here to reflect the new syntax in chains.config
Something like:
```
const configOverwrite = (chains as ConfigHelperConfigOverwrite[]).find(c => c.networkId === config.networkId)
```